src/layout/Group/types.d.ts

```diff
@@ -17,6 +17,7 @@ export interface IGroupEditProperties {
   openByDefault?: boolean | 'first' | 'last';
   alertOnDelete?: ExprVal.Boolean;
   saveAndNextButton?: ExprVal.Boolean;
+  showAddButton?: ExprVal.Boolean;
```

Comment:
Be sure to call it `alwaysShowAddButton` in the implementation as well as the documentation! 😁